software/blinky/main.c

```diff
@@ -0,0 +1,21 @@
+// main.c
+//
+// Copyright (C) 2020 Dan Rodrigues <danrr.gh.oss@gmail.com>
+//
+// SPDX-License-Identifier: MIT
+
+#include <stdint.h>
+#include <stdbool.h>
+
+#include "vdp.h"
+#include "status.h"
+
+int main() {
+    // Blink leds 0..7 along to the vsync clock
+    uint32_t frame;
```

Comment:
Can we init this to 0?  In this demo the reg clearing in `start.S` will probably make `frame` appear zero-initialised on every run, but just in case.

Reply:
Whoops. Yes, of course. Looks like gcc needs `-Wmaybe-uninitialized` to catch this.